src/main/definitions/definition.ts

```diff
@@ -9,3 +9,7 @@ export const enum ClaimOutcomes {
   OLD_JOB = 'oldJob',
   ANOTHER_JOB = 'anotherJob',
 }
+
+export const enum State {
```

Comment:
When user creates a case - state changed would be  AWAITING_SUBMISSION_TO_HMCTS. Can you please double check this ?